docs/alerts/monitors/alert-response.md

```diff
@@ -229,18 +229,39 @@ A _Monitor_ creates an _Alert_. Using the options below, you're subscribing to a
 :::
 
 
-#### From your Alerts list
+### From your Alerts list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row, click the three-dot kebab menu > select **Subscribe**
 * Single-click on a row item > on the opened Alert page, click the three-dot kebab menu > **Subscribe to Monitor**
 
 
-#### From your Monitors list
+### From your Monitors list
 * Right-click on a row item > click **Subscribe**
 * Hover your mouse over a row > click the three-dot kebab menu > click **Subscribe**
 * Single-click on a row item > in the side panel (Monitor Details), click **More Actions** > **Subscribe**
 
 
+## Muting Schedules
+
+:::note
+Make sure you have the **Viewing and Manage Muting Schedules** permissions.
```

Comment:
Hi @angadrandhawa1 , is this the actual name of the user permission?

Reply:
"View Muting Schedules" - this capability/permission is needed for viewing muting schedules page and definitions of schedules.
"Manage Muting Schedules" - this capability/permission is needed for create/edit/delete schedules.